README.md

```diff
@@ -1,2 +1,131 @@
 # node-setup
+
 Connected Cars JavaScript/TypeScript development setup
+
+## Use
+
+``` bash
+npm install https://github.com/connectedcars/node-setup
```

Comment:
Don't we want to publish this on npm so we can do `npm install @connectedcars/setup`?

Reply:
Yes, but it needs a bit more work with including the right files, etc. so this is just the start.